tasks/install_debian.yml

```diff
@@ -1,21 +1,14 @@
 ---
 # tasks file for terraform | Debian/Ubuntu Family
 
-- name: Debian/Ubuntu Family | Downloading archive for {{ terraform_app }} {{ terraform_version }} temporarily to {{ terraform_dl_loc }}
-  ansible.builtin.get_url:
-    url: "{{ terraform_dl_url }}/{{ terraform_app }}/{{ terraform_version }}/{{ terraform_app }}_{{ terraform_version }}_{{ terraform_os }}_{{ terraform_arch }}.zip"
-    dest: "{{ terraform_dl_loc }}"
-
 - name: Debian/Ubuntu Family | Install unzip if it is currently not in installed state
   ansible.builtin.apt:
     name: unzip
     state: present
```

Comment:
@donthorp - Could you please add `update_cache: yes` in the apt install of unzip? This is primarily to address systems that have not fetched repository updates (or don't have index locally) for packages from the repository (such as containers when first initialized)?